### PR TITLE
container registry regenerated

### PR DESCRIPTION
**Pull request by zikalino**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/containerregistry/azure-mgmt-containerregistry/azure/mgmt/containerregistry/v2019_12_01_preview/version.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+VERSION = ""
```
Comment (LianwMS) [inline sdk/containerregistry/azure-mgmt-containerregistry/azure/mgmt/containerregistry/v2019_12_01_preview/version.py]: Does this version need to be updated?

Comment (zikalino) [reply sdk/containerregistry/azure-mgmt-containerregistry/azure/mgmt/containerregistry/v2019_12_01_preview/version.py]: it's irrelevant here, only top level one counts.

**Comment by adewaleo** [comment]
Thanks a lot @zikalino

**Pull request merged**